.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required
 language: java
 jdk:
-  - openjdk7
+  - openjdk8
```

Comment:
Is it possible to test using 2 stages(with jdk8 & jdk11), I wonder if travis support this...
If we don't interagrate the jdk11 part with travis, it's hard to validate every PR works with jdk11

Reply:
Actually I did this PR mainly in two phases, before this PR pass in Travis was compiled both, using Java 11 and then Java 8 like it is observed in this [commit](https://github.com/Intel-bigdata/HiBench/compare/4bbaa5ba08c5a8763b1de92514c0322a472fb702..b20b1f1fc83bf86e25acb8f5d01591649b2538ed), but failed because a timeout issue (_make sense since has to download Java 11 then compile HiBench using it, after this compile in java 8 and so on.._) in Travis side.

However I believe that is possible to make many stages declaring them in the yml file, but don't know if the admin/owner agrees in changing the CI flow.

Reply:
@gczsjdy Travis it's been split in 3 phases (jdk7, jdk8 & jdk11).
I kept jdk7 and basically all related to compile and test on this version to ensure all legacy code like streaming, Kafka etc is not changed and will remain working.